browserify.js

```diff
@@ -1,8 +1,14 @@
-var THREE = require('three');
-var work = require('webworkify');
-var PhysijsWorker = work(require('./physijs_worker.js'));
+class work {
+	constructor(){
+		const work = require('webworkify');
+		return work;
```

Comment:
so this just returns the `webworkify` module and doesn't actually do anything -- it should return `work(require('./physijs_worker.js'))`